run-script.py

```diff
@@ -5,222 +5,234 @@
-			print(f'{column} - Duplicates were found in this database')
-		else:
-			print(f'{column} - Perfect')
-
-	print()
+    '''
+    Input: Pandas dataframe (Participant Database)
+    Output: None - prints if it find duplicates
+    '''
 
-def load_data(private_url, public_url):
-	'''
-	Input: 
-	-------
-	private_url : contains user xslx filepath 
-	public_url  : contains past week partners filepath
+    for column in database.columns:
+        if database[column].duplicated().sum():
+            print(f'{column} - Duplicates were found in this database')
+        else:
+            print(f'{column} - Perfect')
+
+    print()
 
-	Output:
-	-------
-	database    : Pandas dataframe containing participant database
-	register    : Pandas dataframe containing next week participant details
-	weeks       : Contains past week participant details as pandas dataframe
-	'''
 
-	private_db = pd.ExcelFile(private_url)
-	public_db = pd.ExcelFile(public_url)
+def load_data(private_url, public_url):
+    '''
+    Input:
+    -------
+    private_url : contains user xslx filepath
+    public_url  : contains past week partners filepath
 
-	database = private_db.parse('Final')
-	register = private_db.parse('Register')
+    Output:
+    -------
+    database    : Pandas dataframe containing participant database
+    register    : Pandas dataframe containing next week participant details
+    weeks       : Contains past week participant details as pandas dataframe
+    '''
 
-	weeks = [public_db.parse(week) for week in public_db.sheet_names if week.startswith('Week_') ]
+    private_db = pd.ExcelFile(private_url)
+    public_db = pd.ExcelFile(public_url)
 
-	return (database, register, weeks)
+    database = private_db.parse('Final')
+    register = private_db.parse('Register')
 
+    weeks = [public_db.parse(
+        week) for week in public_db.sheet_names if week.startswith('Week_')]
+
+    return (database, register, weeks)
 
 
 def split_partners(week, database):
-	'''
-	Input:
-	-------
-	week           : Contain single week user data
-	database       : Contains entire participant database
-
-	Output:
-	-------
-	partner_list_1 : partner 1 names
-	partner_list_2 : partner 2 names
-	partner_status : status of the participants
-	'''
-
-	def_partners_parse = lambda partners: [participant if participant is not np.nan else None for participant in partners]
-    
-	partner_list_1 = def_partners_parse(week.Partner_1.tolist())
-	partner_list_2 = def_partners_parse(week.Partner_2.tolist())
-	partner_status = (week.Status == 'Started')
-	return (partner_list_1, partner_list_2), partner_status
+    '''
+    Input:
+    -------
+    week           : Contain single week user data
+    database       : Contains entire participant database
+
+    Output:
+    -------
+    partner_list_1 : partner 1 names
+    partner_list_2 : partner 2 names
+    partner_status : status of the participants
+    '''
+
+    def def_partners_parse(partners): return [
```

Comment:
@aravindvnair99, Did you modify this line?

Reply:
Will revert